src/components/layout.js

```diff
@@ -102,164 +99,16 @@ class Layout extends React.Component {
     } else {
       return (
         <DocumentTitle title='Giant Swarm'>
-          <div>
+          <React.Fragment>
```

Comment:
This is new in React 16 and we can start applying it where possible.

`React.Fragment` lets us group components without adding extra unnecessary `<div>`s to the DOM.